doc/book/container/factories.md

```diff
@@ -37,78 +37,78 @@ order to seed the `Application` instance:
 
 - `middleware_pipeline` can be used to seed pre- and/or post-routing middleware:
 
-  ```php
-  'middleware_pipeline' => [
-      // An array of middleware to register prior to registration of the
-      // routing middleware:
-      'pre_routing' => [
-      ],
-      // An array of middleware to register after registration of the
-      // routing middleware:
-      'post_routing' => [
-      ],
+```php
+'middleware_pipeline' => [
+  // An array of middleware to register prior to registration of the
+  // routing middleware:
+  'pre_routing' => [
+  ],
+  // An array of middleware to register after registration of the
+  // routing middleware:
+  'post_routing' => [
   ],
-  ```
+],
+```
 
-  Each item of each array must be an array itself, with the following structure:
+Each item of each array must be an array itself, with the following structure:
 
-  ```php
-  [
-      // required:
-      'middleware' => 'Name of middleware service, or a callable',
-      // optional:
-      'path'  => '/path/to/match',
-      'error' => true,
-  ],
-  ```
+```php
+[
+  // required:
+  'middleware' => 'Name of middleware service, or a callable',
+  // optional:
+  'path'  => '/path/to/match',
+  'error' => true,
+],
+```
 
-  The `middleware` key itself is the middleware to execute, and must be a
-  callable or the name of another defined service. If the `path` key is present,
-  that key will be used to segregate the middleware to a specific matched path
-  (in other words, it will not execute if the path is not matched). If the
-  `error` key is present and boolean `true`, then the middleware will be
-  registered as error middleware. (This is necessary due to the fact that the
-  factory defines a callable wrapper around middleware to enable lazy-loading of
-  middleware.)
```

Comment:
Same thing here; this paragraph is part of the previous bullet point, and that was intentional.